### ISSUE TITLE
UI for the CLI

### ISSUE THREAD

**Issue by vvatelot** [vvatelot/ecoindex_monorepo#8]
Create a simple desktop version using html + CSS

**Comment by PaulPHPE** [comment]
Hello @vvatelot ! 

Do you mean something like swagger but for the CLI ?

**Comment by vvatelot** [comment]
Yes, something more advanced that could be run on desktop whatever os it is. And the idea is just to propose a user friendly version of the cli
